public/src/managers/GRPCManager.ts

```diff
@@ -340,6 +342,14 @@ export class GrpcManager {
         return this.grpcSend<Void>(this.agService.isEmptyRepo, request);
     }
 
+    public getStudentForDiscord(courseCode: string, courseYear: number, userLogin: string) {
```

Comment:
Do you need to implement this method in the frontend??

Reply:
Probably not.